scripts/kata-install/osc-kata-install.sh

```
@@ -157,19 +119,61 @@ install_kata() {
+			[ -f "$conf" ] && cp "$conf" /host/etc/crio/crio.conf.d/
+		done
+
+		# rpm-ostree --apply-live does not run RPM %post scripts on the live
+		# system. The kata-containers %post script normally runs kata-osbuilder
+		# to build a host-kernel-derived kata.kernel/kata.initrd at
+		# /var/cache/kata-containers/osbuilder-images/. Run it manually here
+		# to replicate what the RPM %post would have done.
+		if [[ -x /host/usr/libexec/kata-containers/osbuilder/kata-osbuilder.sh ]]; then
+			echo "Running kata-osbuilder.sh to generate kata VM images..."
+			chroot /host /usr/libexec/kata-containers/osbuilder/kata-osbuilder.sh
+			if [[ $? -eq 0 ]]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)